libvast/builtins/formats/json.cpp

```diff
@@ -555,7 +555,9 @@ class default_parser : public parser_base<FieldValidator> {
         this->ctrl_.warn(caf::make_error(
           ec::parse_error, fmt::format("skips invalid JSON '{}' : {}", view,
```

Comment:
I don't understand how we can feasibly continue here. How can we know where the next JSON object starts?

Reply:
We looked at this together in a call. `stream.truncated_bytes()` must only be called after iterating through the entire stream, which now we're no longer doing. That is, if the documentation of simdjson is correct then this must indeed be a hard-error and not a warning.

Reply:
I didn't find much on this. So it would be safe to just assume we can't call it when the iteration is not complete